src/config.ts

```diff
@@ -115,6 +128,11 @@ const config: ConfigType = {
   zendeskWidgetKey: getEnvironmentVariabel('NDLA_ZENDESK_WIDGET_KEY'),
   localGraphQLApi: getEnvironmentVariabel('LOCAL_GRAPHQL_API', false),
   showAllFrontpageSubjects: true,
+  feideClientID: getEnvironmentVariabel('FEIDE_CLIENT_ID'),
+  feideClientSecret: getEnvironmentVariabel('FEIDE_CLIENT_SECRET'),
+  feideClientIDLocal: getEnvironmentVariabel('FEIDE_CLIENT_ID_LOCAL'),
+  feideClientSecretLocal: getEnvironmentVariabel('FEIDE_CLIENT_SECRET_LOCAL'),
```

Comment:
Hvordan smart er det dette blir gjort andre steder? Håndtering av nøkler/secrets for flere miljøer.

Reply:
Du treng berre ett sett med nøker i koden din. Vi må bruke testnøkler for utviling, og prodnøkler i prod. Dette settes som secrets i aws.